tests/queries/0_stateless/02891_rename_table_without_keyword.reference

```diff
@@ -0,0 +1,7 @@
+r1
+r2
+r3
+test_dictionary
+test_dictionary
```

Comment:
```suggestion
test_dictionary_2
```

So it is okay to rename also dictionaries with `RENAME` as it is also possible with `RENAME TABLE`.

Reply:
OK